src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
```

Comment:
The border-radius value of 0.5px is unusually small and might not have a visible effect. Consider whether this is the intended design or if it should be adjusted.